bridges/XPathBridge.php

```diff
@@ -261,4 +290,57 @@ private function encodeUri($uri)
 
         return $uri;
     }
+
+    protected function setEnclosuresPosition($item)
```

Comment:
Method can be private static.

Reply:
Sorry I'm not familiar with PHP, I don't know how to properly use static keyword

Reply:
```php
class foo {
  private static function bar() {
  }
}
```

Reply:
What I meant is how to call the function afterward.